Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/SettingsTracingFragment.kt

```diff
@@ -117,6 +133,18 @@ class SettingsTracingFragment : Fragment(R.layout.fragment_settings_tracing),
         location.setOnClickListener {
             ExternalActionHelper.toMainSettings(requireContext())
         }
+        interoperability.setOnClickListener {
+            navigateToInteroperability()
```

Comment:
as discussed in https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::32332847-46cb-4995-a3eb-e183b8c68f28?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A21%2C%5C%22sectionId%5C%22%3A22%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3Ac51bf302889a44e28119b0efc59b8f7d%40thread.tacv2%22%7D&tenantId=42f7676c-f455-423c-82f6-dc2d99791af7 please call a method from view model and let live data observer handle navigation